auth/provider_config.go

```diff
@@ -113,11 +117,14 @@ func buildMask(data map[string]interface{}) []string {
 // OIDCProviderConfig is the OIDC auth provider configuration.
 // See https://openid.net/specs/openid-connect-core-1_0-final.html.
 type OIDCProviderConfig struct {
-	ID          string
-	DisplayName string
-	Enabled     bool
-	ClientID    string
-	Issuer      string
+	ID                  string
+	DisplayName         string
+	Enabled             bool
+	ClientID            string
+	Issuer              string
+	ClientSecret        string
+	CodeResponseType    bool
+	IDTokenResponseType bool
```

Comment:
`CodeResponseType` looks to me like it represents a type -- like a string or enum or something. Would `CodeResponseEnabled` etc be clearer?

Reply:
Sure.  Fixed

Reply:
@ScruffyProdigy This changes the approved public API in go/gcip-oidc-provider-code-flow
Could you please check if this requires an amendment to the API proposal?